README.md

```diff
@@ -115,18 +115,17 @@ I want to make JS Paint to be able to edit...
   Images that are partially transparent will automatically open in Transparent mode.
   To enable transparency for an image, go to **Image > Attributes...** or press <kbd>Ctrl+E</kbd>,
   select Transparent, and hit Okay.
-  Then you'll want to remove some of the background.
-  You can use the Eraser tool a bit, then use the Color Picker to
-  pick up where you erased and then use the Fill tool to remove bigger areas.
+  Then you'll want to remove some of the backgrounds.
```

Comment:
There's only one of the background, and I'm talking about some of it, i.e. part of it.